API_Kraken/get_bitcon_price.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import datetime
+
+get_request = requests.get('https://api.kraken.com/0/public/Ticker?pair=XBTUSD')
+price = get_request.json()['result']['XXBTZUSD']['a'][0]
+date = datetime.datetime.now().strftime("%c")
+value = ('Price of bitcon for: ' + date + ' is: ' + price + ' USD')
+try:
```

Comment:
Try should cover all lines of above code especially line 4 which can produce exception.

Move try: to line 3